great_expectations/dataset/pandas_dataset.py

```diff
@@ -160,8 +160,34 @@ class PandasDataSet(MetaPandasDataSet, pd.DataFrame):
     PandasDataset instantiates the great_expectations Expectations API as a subclass of a pandas.DataFrame.
 
     For the full API reference, please see :func:`DataSet <great_expectations.dataset.base.DataSet>`
+
+    NB
+    1. Subsets of PandaDataSet have ALL the expectations of the original
+       data frame unless the user specifies the discard_failing_expectations=True
+       property on the original data frame.
+    2. Samples of PandaDataSet have ALL the non-failing expectations of the original
+       data frame.
+    3. Concatenations, joins, and merges of PandaDataSets ONLY contain the
+       default_expectations (see :func: `add_default_expectations`)
     """
 
+    discard_failed_expectations = False
```

Comment:
Re: line 171: No need to address this in the current PR, but it would be nice to be able to extract sets of expectations from each of the joined datasets, say, and then put them together into the new dataset.

Reply:
Why do samples behave this way? Is something setting discard_failing_expectations?